public/docs/_examples/hierarchical-dependency-injection/ts/src/app/hero-tax-return.component.ts

```diff
@@ -15,10 +15,11 @@ import { HeroTaxReturnService } from './hero-tax-return.service';
 export class HeroTaxReturnComponent {
   message = '';
   @Output() close = new EventEmitter<void>();
-  @Input()
+
   get taxReturn(): HeroTaxReturn {
     return this.heroTaxReturnService.taxReturn;
   }
+  @Input()
```

Comment:
Why did you move this here?  I realize it is technically true that the `@Input` applies to the setter but does it looks awful and it doesn't change Angular's behavior, right?